pepper-apis/config/redisson-jcache-housekeeping.yaml.ctmpl

```diff
@@ -0,0 +1,28 @@
+{{with $environment := env "ENVIRONMENT"}}
+{{with $version := env "VERSION"}}
+{{with $conf := vault (printf "secret/pepper/%s/%s/conf" $environment $version)}}
+---
+singleServerConfig:
+  idleConnectionTimeout: 10000
+  connectTimeout: 5000
+  timeout: 3000
+  retryAttempts: 3
+  retryInterval: 1500
+  password: null
+  subscriptionsPerConnection: 5
+  clientName: null
+  address: "redis://{{$conf.Data.redis.host}}:{{$conf.Data.redis.port}}"
+  subscriptionConnectionMinimumIdleSize: 1
+  subscriptionConnectionPoolSize: 50
+  connectionMinimumIdleSize: 24
+  connectionPoolSize: 64
+  database: 0
+  dnsMonitoringInterval: 5000
+  pingConnectionInterval: 15000
+threads: 16
+nettyThreads: 32
```

Comment:
Went down from 32 to 16 and from 128 to 32. These new values are the default ones.